server/src/db/migrations/20220119232016_store-front.ts

```diff
@@ -0,0 +1,123 @@
+import { Knex } from "knex";
+
+
+export async function up(knex: Knex): Promise<void> {
+
+    knex.schema.hasTable('InventoryItem').then(function (exists) {
+        if (!exists) {
+            return knex.schema.createTable('InventoryItem', function (t) {
+                t.increments('id').primary();
+                t.text('image');
+                t.text('name');
+                t.text('unit');
+                t.text('pluralUnit');
+                t.integer('count');
+                t.float('pricePerUnit');
+            });
+        }
+    });
+
+    // if we know every label we can use an enum instead of this table
+    knex.schema.hasTable('Label').then(function (exists) {
```

Comment:
Created a separate table instead of enum so that end-users can create their own specific labels. If we can think of every label they would ever need it might be better to just do an enum...

Reply:
probably should add a way for people to create labels tho...